r/tests/testthat/test-dplyr-funcs-datetime.R

```diff
@@ -1600,17 +1600,20 @@ test_that("`as_datetime()`", {
     double_date = c(10.1, 25.2, NA)
   )
 
-  test_df %>%
-    arrow_table() %>%
-    mutate(
-      ddate = as_datetime(date),
-      dchar_date_no_tz = as_datetime(char_date),
-      dchar_date_non_iso = as_datetime(char_date_non_iso, format = "%Y-%d-%m %H:%M:%S"),
-      dint_date = as_datetime(int_date, origin = "1970-01-02"),
-      dintegerish_date = as_datetime(integerish_date, origin = "1970-01-02"),
-      dintegerish_date2 = as_datetime(integerish_date, origin = "1970-01-01")
-    ) %>%
-    collect()
+  compare_dplyr_binding(
```

Comment:
I expect some tests will fail, mostly due to the discarding of the characters before the `"_"` during the registration of the binding. If we switch to the `"package_fun_name"` naming convention, that should be fine.